src/commerce-handler.js

```diff
@@ -75,7 +75,8 @@ CommerceHandler.prototype.logCommerceEvent = function(event) {
             }
         }
 
-        if (optimizelyFullStackEvents.events[expandedEvent.EventName]) {
+        // if optimizely full stack is being used
+        if (window.optimizelyClientInstance) {
```

Comment:
Not part of your code, but in lines 14-76, let's rename the `optimizelyEvents` to `optimizelyWebXEvents` and `optimizelyEvent` to `optimizelyWebXEvent` to be more clear. as well as rename the module `optimizely-defined-events.js` to `optimizely-x-defined-events.js`